modules/ReduxAsyncConnect.js

```diff
@@ -43,7 +43,7 @@ function asyncConnectPromises(components, params, store, helpers) {
 export function loadOnServer({ components, params }, store, helpers) {
   return Promise.all(asyncConnectPromises(filterAndFlattenComponents(components), params, store, helpers))
     .catch(error => console.error('reduxAsyncConnect server promise error: ' + error)).then(() => {
-      store.dispatch(initialDataLoaded());
+      store.dispatch(endLoad());
```

Comment:
@YAndrew91 thanks, fixed. Going to write tests to avoid such problems in future.